ipa-core/src/hpke/mod.rs

```diff
@@ -26,7 +26,7 @@ use crate::{
 
 /// IPA ciphersuite
 type IpaKem = hpke::kem::X25519HkdfSha256;
-type IpaAead = hpke::aead::AesGcm128;
+pub type IpaAead = hpke::aead::AesGcm128;
```

Comment:
I think that is what is causing issues now - you're using AES-GCM-256 on Swift side and switched to AES-GCM-128. I don't know the reason, but I think it needs to be consistne

Reply:
This did it! This was the last one remaining!!!